src/components/BankTransactionsList/BankTransactionsProcessingInfo.tsx

```diff
@@ -1,12 +1,34 @@
 import { BookkeepingPeriodStatus } from '@hooks/bookkeeping/periods/useBookkeepingPeriods'
+import Clock from '@icons/Clock'
+import { HStack } from '@ui/Stack/Stack'
+import { Badge, BadgeSize } from '@components/Badge/Badge'
 import { BookkeepingStatus } from '@components/BookkeepingStatus/BookkeepingStatus'
 import { DeprecatedTooltip, DeprecatedTooltipContent, DeprecatedTooltipTrigger } from '@components/Tooltip/Tooltip'
 
-export const BankTransactionsProcessingInfo = () => (
-  <DeprecatedTooltip offset={12}>
-    <DeprecatedTooltipTrigger><BookkeepingStatus status={BookkeepingPeriodStatus.IN_PROGRESS_AWAITING_BOOKKEEPER} text='Processing' /></DeprecatedTooltipTrigger>
-    <DeprecatedTooltipContent className='Layer__tooltip' width='md'>
-      {'Our team will review and categorize this transaction. We\'ll reach out if we have any questions about it.'}
-    </DeprecatedTooltipContent>
-  </DeprecatedTooltip>
-)
+import './bankTransactionsProcessingInfo.scss'
+interface BankTransactionsProcessingInfoProps {
+  showAsBadge?: boolean
+}
+
+export const BankTransactionsProcessingInfo = ({ showAsBadge = false }: BankTransactionsProcessingInfoProps) => {
+  const tooltipContent = 'Our team will review and categorize this transaction. We\'ll reach out if we have any questions about it.'
+
+  if (showAsBadge) {
+    return (
+      <HStack gap='xs' align='center' fluid className='Layer__BankTransactionsProcessingInfo'>
+        <Badge size={BadgeSize.SMALL} icon={<Clock size={11} />}>
```

Comment:
In the PR screenshot the badge looks purple? But I don't see any change that would actually cause that here.

I _do_ actually care about keeping it that shade of blue because we use that same color elsewhere in the product (`<Tasks>`) to indicate it's being worked on by bookkeepers so I want to keep that design language the same.